lib/ansible/modules/system/listen_ports_facts.py

```diff
@@ -20,64 +20,42 @@
 
 DOCUMENTATION = '''
 ---
-module: portenforce
+module: listen_ports_facts
 author:
     - '"Nathan Davison (@ndavison)" <ndavison85@gmail.com>'
 version_added: "2.3"
 description:
-    - Kill processes listening on TCP and UDP ports not defined in the whitelists.
-short_description: Kill processes listening on TCP and UDP ports not defined in the whitelists.
+    - Gather facts on processes listening on TCP and UDP ports. Optionally provide 
+    a whitelist of TCP and/or UDP ports to gather facts on processes that violate 
+    the whitelists.
+short_description: Gather facts on processes listening on TCP and UDP ports.
 options:
   whitelist_tcp:
     description:
-      - A list of TCP ports to allow.
+      - A list of TCP port numbers that are expected to have processes listening.
     required: false
     type: "list"
   whitelist_udp:
     description:
-      - A list of UDP ports to allow.
+      - A list of UDP port numbers that are expected to have processes listening.
     required: false
     type: "list"
 '''
 
 EXAMPLES = '''
-- name: only allow TCP 80, 443, and 22
```

Comment:
I believe the example now is showing how to use the returned data - it's based on the `ec2_facts` module but with a loop.